files/rsyslog/60-mmk8s.conf

```diff
@@ -17,31 +17,31 @@ if ((strlen($!CONTAINER_NAME) > 0) and (strlen($!CONTAINER_ID_FULL) > 0)) or
             action(type="mmnormalize" rulebase="/etc/rsyslog.d/multiline-json.rulebase")
             foreach ($.ii in $!multilinejson) do {
                 if strlen($!@timestamp) == 0 then {
```

Comment:
should this be `$.all!@timestamp` now? Same with `$.all!stream` and `$.all!log` below.  and in the crio section

Reply:
Ah, you are right.  I misunderstood the logic... :p  Fixing them.  Thanks!